### ISSUE TITLE
Add support for RequireJS 2.2.x

### ISSUE THREAD

**Issue by kellym** [karma-runner/karma-requirejs#41]
The current version has requirejs locked to `~2.1`. RequireJS 2.2.0 is out -- looks like no breaking changes. http://jrburke.com/2016/03/17/requirejs-2.2-alameda-1.0-released/

**Comment by biril** [comment]
+1 for this one. Our build is failing since this morning, on npm 2.14.20 with

```
npm ERR! peerinvalid The package requirejs@2.2.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer karma-requirejs@0.2.5 wants requirejs@~2.1

```

**Comment by jbrunel** [comment]
+1, build is failing on our side as well. I created pull request #43.

**Comment by LaurentMox** [comment]
+1, this is critical for our builds :cry:

**Comment by dignifiedquire** [comment]
Fixed in 0.2.6
